src/libraries/System.Private.Runtime.InteropServices.JavaScript/src/System/Runtime/InteropServices/JavaScript/Runtime.cs

```diff
@@ -323,6 +323,10 @@ private static string GetCallSignature(IntPtr methodHandle)
                         {
                             res[c] = 'h';
                         }
+                        else if (t == typeof(string[]))
+                        {
+                            res[c] = 'm';
+                        }
```

Comment:
Is this change specific to entry point methods? Can we add tests for this?

Reply:
It isn't